client/src/test/java/com/sinch/sdk/domains/numbers/adapters/AvailableNumberServiceTest.java

```diff
@@ -80,37 +59,48 @@ void list() throws ApiException {
             .build();
 
     Collection<AvailableNumber> expected =
-        Collections.singletonList(
-            AvailableNumber.builder()
-                .setPhoneNumber("+46650553763")
-                .setRegionCode("SE")
-                .setType(NumberType.LOCAL)
-                .setCapability(Collections.singletonList(Capability.VOICE))
-                .setSetupPrice(new Money("DOLLAR", 0.57))
-                .setMonthlyPrice(new Money("EUR", 0.80))
-                .setPaymentIntervalMonths(1)
-                .setSupportingDocumentationRequired(true)
-                .build());
+        new ArrayList<>(
+            Collections.singletonList(
+                AvailableNumber.builder()
+                    .setPhoneNumber("+46650553763")
+                    .setRegionCode("SE")
+                    .setType(NumberType.LOCAL)
+                    .setCapability(new ArrayList<>(Collections.singletonList(Capability.VOICE)))
+                    .setSetupPrice(new Money("DOLLAR", 0.57))
```

Comment:
Used for unit test and because of a free form text, it should not hurt 😃 
Used values could be enhanced in the future if we need to stick to more real use case values